assets/js/app/core/directives/RawView.js

```diff
@@ -34,6 +37,14 @@
                 '</div>',
                 controller: function($scope,$uibModalInstance,_item){
                   $scope.item = _item
+                  $scope.data = 'data:text/plain;charset=utf-8,' +
```

Comment:
Should this be data:application/json?

Reply:
Doesn't seem to have an effect on functionality, but it would be clearer. Done.